tools/ote_sample.py

```diff
@@ -49,11 +42,15 @@ def parse_args():
     parser.add_argument('template_file_path', help='path to template file')
     parser.add_argument('--data-dir', default='data')
     parser.add_argument('--export', action='store_true')
+    parser.add_argument('--debug-dump-path', default='')
     args = parser.parse_args()
     return args
 
 
 def main(args):
+    if args.debug_dump_path:
+        from torchreid.utils import Logger
```

Comment:
Could you, please, add a log name with a timestamp for the log file:

`log_name = 'ote_task.log'`
`log_name += time.strftime('-%Y-%m-%d-%H-%M-%S')`